testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/oid4vc/issuance/signing/JwtSigningServiceTest.java

```diff
@@ -53,7 +52,7 @@
 import static org.junit.Assert.fail;
 
 
-public class JwtSigningServiceTest extends SigningServiceTest {
+public class JwtSigningServiceTest extends OID4VCTest {
```

Comment:
L76 and L78 are opposite.

Reply:
Amazing, how are you able to spot such thing!?:) Fixed in [42f5903](https://github.com/keycloak/keycloak/pull/27931/commits/42f5903f054b931386c5c249d4f2990c99ed4673)

Reply:
Thank you.